include/lexer.h

```diff
@@ -0,0 +1,40 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        ::::::::            */
+/*   lexer.h                                            :+:    :+:            */
+/*                                                     +:+                    */
+/*   By: mbp14 <mbp14@student.codam.nl>               +#+                     */
```

Comment:
You can change email + username used in header in the VSCode extension settings

Reply:
done